content/blog/2020-09-22-treedata.table/index.md

```diff
@@ -0,0 +1,82 @@
+
+Our other two vignettes cover important data functions researchers may wish to do with phylogenetic data.
+For example, syncing names across datasets and phylogenetic trees based on [partial matching](https://ropensci.github.io/treedata.table/articles/C_PartialMatching.html), and [assorted data tidying functions](https://ropensci.github.io/treedata.table/articles/D_AdditionalFunctions_treedata.table.html).
+
+### Where did treedata.table come from?
+<div style="text-align:center">
+<img src='img/NantucketBeach.jpg' alt="Beach with small waves" align="middle"width="600" />
+</div>
+<font size="2"> **Fig. 2.** The beach outside the workshop bunkhouse. </font>
+
+treedata.table is a research output from the [Nantucket DevelopR workshop](https://github.com/NantucketDevelopeR/2019Workshop).
+Funded by an NSF grant to [Liam Revell](http://www.faculty.umb.edu/liam.revell/), the workshop aimed to support intermediate R developers working with phylogenetic and phylogenetic comparative methods.
+Last time around, we had four instructors ([April Wright](https://paleantology.com), [Josef Uyeda](http://www.uyedalab.com/), [Claudia Solis-Lemus](https://solislemuslab.github.io/) and [Klaus Schliep](https://www.phangorn.org/)).
+Each instructor taught one serious lesson during the week, and most did one "fireside chat" about being a computational scientist working in R.
+
+But the real heart of the workshop was spending the day immersed in coding, both collaboratively and alone.
+Participants each came with a project in mind, and made pitches to others for their project.
+Some participants worked on multiple projects, or one with a group and one alone.
+The instructors were housed separately, and often we'd show up to the beachside house to find everyone didn't want lecture yet - they were too busy working already.
+Intermediate programmers are often sort of left to their own devices in education, and spending a week developing skills, community, and friendships was absolutely lovely. And, as evidenced by this package, productive!
+
+### Acknowledgements
+
+We'd like to thank all the participants at the workshop for a very special week together in an amazing place.
+We would also like to thank the instructors (Claudia and Klaus), Liam (NSF DBI-1759940), UMass field station coordinator Yolanda Vaillancourt, and course facilitator Fabio Machado Lugar. We would also like to thank package reviewers [Hugo Gruson](https://github.com/Bisaloo), [Kari Norman](https://github.com/karinorman), editor [Julia Gustavson](https://github.com/jooolia) and blog editor [Steffi LaZerte](https://github.com/steffilazerte). AMW was supported by an Institutional Development Award (IDeA) from the National Institute of General Medical Sciences of the National Institutes of Health under grant number P2O GM103424-18.
```

Comment:
Thank you! (but you really don't have to). 

I've added rOpenSci links to authors where they exist and a link to the review thread.

```suggestion
We would also like to thank the instructors (Claudia and Klaus), Liam (NSF DBI-1759940), UMass field station coordinator Yolanda Vaillancourt, and course facilitator Fabio Machado Lugar. treedata.table is an rOpenSci [peer-reviewed package](https://github.com/ropensci/software-review/issues/367). We would also like to thank package reviewers [Hugo Gruson](/author/hugo-gruson/), [Kari Norman](/author/kari-norman/), editor [Julia Gustavson](https://github.com/jooolia) and blog editor [Steffi LaZerte](/author/steffi-lazerte). AMW was supported by an Institutional Development Award (IDeA) from the National Institute of General Medical Sciences of the National Institutes of Health under grant number P2O GM103424-18.
```